aws-redshiftserverless-namespace/src/main/java/software/amazon/redshiftserverless/namespace/Translator.java

```diff
@@ -64,6 +66,26 @@ static List<software.amazon.awssdk.services.redshiftserverless.model.Tag> transl
             .collect(Collectors.toList());
   }
 
+  /*
+    This function is to return the iam role in the same format as input iam roles.
+    Instead of modifying the schema for backward compatibitlity we use regex to extract the iam role.
+    'IamRole(applyStatus=null, iamRoleArn=arn:aws:iam::254260483320:role/contracttest9nfdg-redshif-RedshiftServerlessNamespa-nAxEywsQousB)'
+   */
+  static List<String> translateIamRoles(final List<String> iamRoles) {
+    return Optional.ofNullable(iamRoles).orElse(Collections.emptyList())
+            .stream()
+            .map(iamRole -> {
+              Pattern iamPattern = Pattern.compile("arn:aws(-cn|-us-gov)?:iam::([0-9]{12})?:role/.+$");
```

Comment:
can we do something like this, or ArnBuilder to get the arn string? we won't be able to cover iso regions and whatever new AWS regions partition this way

import software.amazon.awssdk.regions.Region;

public class AwsHelper {

    public static String getPartitionFromRegion(String regionName) {
        Region region = Region.of(regionName);
        return region.metadata().partition().id();
    }